### PR TITLE
[16.0] [MIG] asterisk_click2dial: Migration to 16.0

### PR DESCRIPTION
**Pull request by luisDIXMIT**
Standard migration.

@etobella

### CODE REVIEW

**Review by etobella — APPROVED** [review]
Tested locally

File: asterisk_click2dial/models/asterisk_server.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class AsteriskServer(models.Model):
     ip_address = fields.Char(string="Asterisk IP address or DNS", required=True)
     port = fields.Integer(
         required=True,
-        default=5038,
+        default=8088,
```
Comment (cvinh) [inline asterisk_click2dial/models/asterisk_server.py:28]: default port for AMI is 5038 indeed
8088 is for the web interface

File: asterisk_click2dial/models/asterisk_server.py [hunk 2]
```diff
@@ -146,29 +146,33 @@ def _get_connect_info(self, url_path):
...
-                "the Asterisk Manager Interface."
-            )
-        )
+
+        return {
+            "type": "ir.actions.client",
+            "tag": "display_notification",
+            "params": {
+                "type": "success",
+                "message": _("Your request has been successfully sent"),
```
Comment (cvinh) [inline asterisk_click2dial/models/asterisk_server.py]: Why did you change the message ?
IMHO, the original one is better because the connexion is towards ARI

**Review by cvinh — CHANGES_REQUESTED** [review]

File: asterisk_click2dial/models/asterisk_server.py [hunk 3]
```diff
@@ -146,29 +146,33 @@ def _get_connect_info(self, url_path):
...
 
     def test_ari_connection(self):
         self.ensure_one()
         auth = (self.login, self.password)
-        url = "http://%s:%s/ari/asterisk/info" % (self.ip_address, self.port)
+        url = f"http://{self.ip_address}:{self.port}/ari/asterisk/info"
         try:
             res = requests.get(url, auth=auth, timeout=TIMEOUT)
         except Exception as e:
             raise UserError from None(
```
Comment (cvinh) [inline asterisk_click2dial/models/asterisk_server.py]: I have models/asterisk_server.py", line 159, in test_ari_connection
TypeError: 'NoneType' object is not callable

File: asterisk_click2dial/models/asterisk_server.py (hunk 1 above)
Comment (etobella) [reply asterisk_click2dial/models/asterisk_server.py:28]: AMI is no longer used. It was changed on v14

Comment (cvinh) [reply asterisk_click2dial/models/asterisk_server.py:28]: OMG, time goes fast

File: asterisk_click2dial/models/asterisk_server.py (hunk 3 above)
Comment (luisDIXMIT) [reply asterisk_click2dial/models/asterisk_server.py]: Thanks for your review, comments attended!

**Review by cvinh — APPROVED** [review]

**Review by nimarosa — APPROVED** [review]

**Comment by nimarosa** [comment]
Tested and working ok. Merging. 

/ocabot merge nobump
